CalibCalorimetry/HcalAlgos/src/HcalPulseShapes.cc

```diff
@@ -268,6 +270,283 @@ void HcalPulseShapes::computeHFShape() {
 }
 
 
+void HcalPulseShapes::computeSiPMShapeLAG2017()
+{
+
+   //Combination of all phase scan data (May,Jul,Oct2017)
+   //L(X)AG
+   //runs:  294736-294740, 294929-294950, 298594-298598 and 305744-305758
+
+  unsigned int nbin = 250;
+
+  std::vector<float> nt = {
+	5.22174e-12,
+	7.04852e-10,
+	3.49584e-08,
```

Comment:
please wrap multiple entries on one line (e.g. 10)
also,  can this  be changed to an std::array?

Reply:
All the three related templates are changed to type std::array with multiple entries per line. Please check. Let me know if you have any comment.